packages/pyright-internal/src/languageService/autoImporter.ts

```diff
@@ -279,11 +279,11 @@ export class AutoImporter {
 
             if (
                 !isStubOrHasInit.isStub &&
-                autoImportSymbol.kind === CompletionItemKind.Variable &&
-                /[a-z]/.test(name)
+                (autoImportSymbol.kind === undefined || autoImportSymbol.kind === CompletionItemKind.Variable) &&
```

Comment:
which Symbol Kind is "autoImportSymbol.kind === undefined" for?

Reply:
I am asking since now it will check every symbol rather than only variable symbol kind.

Reply:
I think what you want is tweaking this (https://github.com/microsoft/pyright/blob/d5b7725a6f60248d515e1541a5674f9f487443e0/packages/pyright-internal/src/languageService/autoImporter.ts#L94), not kind === undefined.

Reply:
exiting code basically says, if pyright already figured out that variable is "Final" or "Const", then we will always show it in auto-import regardless of how the name is written.

Reply:
I see. Originally I noticed that protected constants where also showing up, which is why I added that check. However after you mentioning it, I now believe that it might be for the better the way constants are currently handled. Unlike with variables, there are at least some scenarios in which you might what to import them.

I'll change it.